static/app/views/replays/detail/console/index.tsx

```diff
@@ -26,6 +27,11 @@ interface Props {
 function Console({breadcrumbs, startTimestampMs = 0}: Props) {
   const {currentHoverTime, currentTime} = useReplayContext();
   const containerRef = useRef<HTMLDivElement>(null);
+
+  useScrollPosition({
+    scrollContainerRef: containerRef,
+    id: 'CONSOLE',
+  });
```

Comment:
Subscribing to this context is probably gonna cause a lot of re renders to the tab contents